CMakeLists.txt

```diff
@@ -1,5 +1,14 @@
 cmake_minimum_required(VERSION 3.14)
-cmake_policy(SET CMP0095 OLD)  # RPATH escaping
+
+# RPATH escaping (only in old CMake versions)
+if (CMAKE_VERSION VERSION_LESS "3.24.0")
+	cmake_policy(SET CMP0095 OLD)
+endif()
```

Comment:
We should fix that in the RPATH itself. This will break some installs for CMake >= 3.24